src_blog/_posts/Grain-Formatter.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Introducing the Grain Formatter
+subtitle: Bringing consistency and pretty code to Grain
+date: 2021-08-31 12:00:00
+author: Marcus Roberts
+cover: /blog/photos/cover/aaron-burden-FGbLYvTgxx0-unsplash.jpg
+coverAttribution: AAron Burden / Unsplash
+tags: Tooling, Developer Experience, 
+---
+
+For any modern programming language to succeed it needs to provide developers the support tools they have become used to: code editor extensions, IDE support and in recent years code formatting or pretty print tools.
```

Comment:
```suggestion
For any modern programming language to succeed, it needs to provide developers the support tools they have become used to: code editor extensions, IDE support and in recent years code formatting or pretty print tools.
```
Not sure if this was the comma war you were talking about, so I'll come with a citation :joy:
https://www.grammarly.com/blog/subordinate-clause/
> When a subordinate clause begins a sentence, it has a comma after it.